### DISCUSSION TITLE
Change coreWebReleaseVersion and webComponentsReleaseVersion to correct version for 22.01 branch

### DISCUSSION THREAD

**Discussion by freddyucv** [dotCMS/core#21611] [Ideas]
Issue: https://github.com/dotCMS/core/issues/21603

@dsilvam 
@victoralfaro-dotcms

**Comment by freddyucv** [comment]
@victoralfaro-dotcms What this mean:

"Currently coreWebReleaseVersion and webComponentsReleaseVersion properties at core's gradle.properties file are pointing to rc. They should be pointing to 22.1.0-rc.1 instead."

what is rc or 22.1.0-rc.1?

Also

"This is done through the release process workflow which is failing to update so. Therefore this issue is contemplating to fix this dot-cicd bug."

what is the dot-cicd bug?

**Reply by victoralfaro-dotcms** [reply]
`22.1.0-rc.1` is the version which `release-22.01` branch's `coreWebReleaseVersion` and `webComponentsReleaseVersion` properties (at gradle.properties) should point to. Right now is pointing to `rc`.
This happened because the automated release process failed in doing the action of changing those values. This is a `dot-cicd` bug.
